### PR TITLE
Update DBT action and DBT github workflow [ch3433]

### PR DESCRIPTION
**Pull request by alyiwang**
Add "account" parameter in DBT action

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,13 +12,17 @@ Add the following to the [GitHub Workflow](https://docs.github.com/en/actions/re
   with:
         manifest: metaphor/target/manifest.json
         catalog: metaphor/target/catalog.json
+        account: <account_number>
```
Comment (mars-lan) [inline README.md:15]: See https://github.com/MetaphorData/connectors/pull/29#discussion_r701802269

Comment (alyiwang) [reply README.md:15]: The comment has been resolved

File: README.md [hunk 2]
```diff
@@ -12,13 +12,17 @@ Add the following to the [GitHub Workflow](https://docs.github.com/en/actions/re
   with:
         manifest: metaphor/target/manifest.json
```
Comment (mars-lan) [inline README.md:13]: Please fix these path.

Comment (mars-lan) [reply README.md:13]: Actually seems like these are copy & paste errors. Let 's drop Line 13-16

**Review by mars-lan — APPROVED** [review]

**Pull request merged**